CHANGELOG.md

```diff
@@ -5,6 +5,9 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 ## [Unreleased]
 ### Added
 
+### Fixed
+- Fix detection of path when using nested properties in ``AdminHelper:getElementAccessPath`` method
```

Comment:
I have a better understanding of your issue since: https://github.com/sonata-project/SonataAdminBundle/issues/3763#issuecomment-217808893

So this fix issue for underscore cased properties. Could you please add this precision?